docs/UserGuide.md

```diff
@@ -53,36 +53,14 @@ Examples:
 
 ### Find pet Id: `find`
```

Comment:
I think it will be better to rename **Find pet Id:** to a more appropriate name. Maybe **Find pet details:** ?